dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -26,17 +26,74 @@ class ScriptLevelsControllerTest < ActionController::TestCase
     @custom_s2_l2 = create(:script_level, script: @custom_script,
                            stage: @custom_stage_2, :position => 2)
     client_state.reset
+
+    Gatekeeper.clear
+    DCDO.clear
```

Comment:
I don't think this should be necessary, In test_helper.rb I have it automatically clear Gatekeeper and DCDO for each test.